src/UNL/MediaHub/DefaultHomepage.php

```diff
@@ -22,12 +22,54 @@ function getCacheKey()
 
     function run()
     {
+        
+    }
+    
+    function getTopMedia()
+    {
         $options = array(
-            'limit'   => 6,
+            'limit'   => null,
             'orderby' => 'popular_play_count'
         );
-        $this->top_media = new UNL_MediaHub_MediaList($options);
-        $this->top_media->run();
+        $top_media = new UNL_MediaHub_MediaList($options);
```

Comment:
I'm just pointing out an opportunity for optimization. In either case, if you are not longer using the `top_media` member, please remove it from the class.